### PR TITLE
General UI bug fixes for version 0.6.1

### PR DESCRIPTION
**Pull request by edwardcqian**
- Fixed reset button for OpenAI Key field
- Fixed an issue not allowing openai provider option to be saved
- Fixed local dev for cloud instances by adding a `DEV_MODE` environment variable

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -15,6 +15,7 @@ services:
       VECTOR_EMBEDDER_BASIC_AUTH_PASSWORD: $VECTOR_EMBEDDER_BASIC_AUTH_PASSWORD
       VECTOR_STORE_BASIC_AUTH_PASSWORD: $VECTOR_STORE_BASIC_AUTH_PASSWORD
       GF_FEATURE_TOGGLES_ENABLE: externalServiceAccounts
+      NOT_HG: true
```
Comment (csmarchbanks) [inline docker-compose.yaml:18]: Maybe DEV_MODE or something instead? The double negative can be a bit confusing, and more accurately represents the state.

**Comment by edwardcqian** [comment]
Tested in dev, seems like the current bugs are fixed.

**Comment by gerboland** [comment]
You should explain what bugs you're fixing, so the reviewer can reproduce the issue and validate the fix.

**Review by gerboland — APPROVED** [review]

**Comment by edwardcqian** [comment]
> You should explain what bugs you're fixing, so the reviewer can reproduce the issue and validate the fix.

Added them to the ticket, forgot to actually put them in the PR 😅

**Pull request merged**